src/udev/udevd.c

```diff
@@ -1035,6 +1035,7 @@ static int on_ctrl_msg(sd_event_source *s, int fd, uint32_t revents, void *userd
         if (i >= 0) {
                 log_debug("Received udev control message (SET_LOG_LEVEL), setting log_priority=%i", i);
                 log_set_max_level(i);
+                log_set_max_level_realm(LOG_REALM_SYSTEMD, i);
```

Comment:
Hmm, can you describe (in the commit message) what the effect of this is? I think I understand, but it's easier to review the patch if the *intent* is clear.

Reply:
OK. I will update the commit message. Sorry for my laziness.

The reason is, now not a few `udevd` debug logs are come from sd-device or sd-hwdb. Only setting `LOG_REALM_UDEV` may not sufficient to debug. We already have similar code in `main()` and `udevadm`.